code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -0,0 +1,207 @@
+/obj/item/weapon/nullrod
+	name = "null rod"
+	desc = "A rod of pure obsidian, its very presence disrupts and dampens the powers of Nar-Sie's followers."
+	icon_state = "nullrod"
+	item_state = "nullrod"
+	force = 18
+	throw_speed = 3
+	throw_range = 4
+	throwforce = 10
+	w_class = 1
+	var/reskinned = FALSE
+
+/obj/item/weapon/nullrod/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is killing \himself with \the [src.name]! It looks like \he's trying get closer to god!</span>")
+	return (BRUTELOSS|FIRELOSS)
+
+/obj/item/weapon/nullrod/attack_self(mob/user)
+	if(reskinned)
+		return
+	if(user.mind && (user.mind.assigned_role == "Chaplain"))
+		reskin_holy_weapon(user)
+
+/obj/item/weapon/nullrod/proc/reskin_holy_weapon(mob/M)
+	var/list/holy_weapons_list = typesof(/obj/item/weapon/nullrod)
+	var/list/display_names = list()
+	for(var/V in holy_weapons_list)
+		var/atom/A = V
+		display_names += initial(A.name)
+
+	var/choice = input(M,"What theme would you like for your holy weapon?","Holy Weapon Theme") as null|anything in display_names
+	if(!src || !choice || M.stat || !in_range(M, src) || M.restrained() || !M.canmove || reskinned)
+		return
+
+	var/index = display_names.Find(choice)
+	var/A = holy_weapons_list[index]
+
+	var/obj/item/weapon/nullrod/holy_weapon = new A
+
+	feedback_set_details("chaplain_weapon","[choice]")
+
+	if(holy_weapon)
+		holy_weapon.reskinned = TRUE
+		M.unEquip(src)
+		M.put_in_active_hand(holy_weapon)
+		qdel(src)
+
+/obj/item/weapon/nullrod/godhand
+	icon_state = "disintegrate"
+	item_state = "disintegrate"
+	name = "god hand"
+	desc = "This hand of yours glows with an awesome power!"
+	flags = ABSTRACT | NODROP
+	w_class = 5
+	hitsound = 'sound/weapons/sear.ogg'
+	damtype = BURN
+	attack_verb = list("punched", "cross countered", "pummeled")
+
+/obj/item/weapon/nullrod/staff
+	icon_state = "godstaff-red"
+	item_state = "godstaff-red"
+	name = "red holy staff"
+	desc = "It has a mysterious, protective aura."
+	w_class = 5
+	force = 5
+	slot_flags = SLOT_BACK
+	block_chance = 50
+	var/shield_icon = "shield-red"
+
+/obj/item/weapon/nullrod/staff/worn_overlays(isinhands)
+    . = list()
+    if(isinhands)
+        . += image(icon = 'icons/effects/effects.dmi', icon_state = "[shield_icon]")
```

Comment:
Isn't this not supposed to compile because of inconsistent indentation
Fuckin dm

My bad tho